worker/pkg/workflows/datasync/activities/sync/activity_test.go

```diff
@@ -191,7 +191,7 @@ pipeline:
   threads: 1
   processors:
     - mutation: |
-        root.name = fake("first_name")
+        root.name = generate_first_name()
```

Comment:
This is also a drive by change. I was attempting to replace bento wtih benthos core and it doesn't look like `fake` is available anymore. We don't expose this to end-users anyways so I am updated this to be one of our functions.